src/internal/m365/collection/groups/collection.go

```diff
@@ -280,7 +291,8 @@ func (col *Collection) streamItems(ctx context.Context, errs *fault.Bus) {
 
 			// TODO: the item getter should provide this func when we switch
 			// to a lookup-id-then-item pattern.
-			info := api.ChannelMessageInfo(item, int64(len(data)), col.chanID, col.chanName)
+			info.ChannelID = col.chanID
+			info.ChannelName = col.chanName
```

Comment:
Are we going to store teams info on meta data level too? That would help us in creating teams it that was also deleted.

Reply:
we have a couple different types of metadata.  Did you mean alongside previous paths and delta tokens? Or did you mean as a `foo.meta` item?  If the latter (.meta), I'd like to avoid that unless absolutely necessary.  And since we're not planning to restore teams or channels, we might not need that info.